Client/AdjustHelper.swift

```diff
@@ -90,5 +100,15 @@ extension AdjustHelper: AdjustDelegate {
         if !shouldEnable {
             AdjustHelper.setEnabled(false)
         }
+
+        telemetryHelper.setAttribution(attribution)
+    }
+
+    func adjustDeeplinkResponse(_ deeplink: URL?) -> Bool {
```

Comment:
Just to confirm I understand, each time we receive the deeplink we send the telemetry that we set earlier on the setup ?

Reply:
no, according to the brief there are 2 steps needed for this story 1 setup Adjust attribution and it should be send in the metrics ping and add an event when a deeplink is received in Adjust callback. This is for the second case